code/game/objects/items/plushes.dm

```diff
@@ -662,8 +662,7 @@
 	attack_verb_continuous = list("flutters", "flaps")
 	attack_verb_simple = list("flutter", "flap")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
-///Used to track how many people killed themselves with item/toy/plush/moth
-	var/suicide_count = 0
+	hide_suicide = TRUE
```

Comment:
Having the `hide_suicide` variable, in your code, immediately returns on `examine()` and _does not_ show the number of people who killed themselves. According to the comment from the old plushes.dm:

```dm
///Used to track how many people killed themselves with item/toy/plush/moth
```

Your current code makes it such that those who suicide with `obj/item/toy/plush/moth` never shows the number of people who kill themselves on it. Is this intentional?

Reply:
yea, because moth toy updates the description based on the number of suicide, it would be weird to have that display multiple times

Reply:
might be worthwhile to comment that in then just so people are aware of that nichety that moth plushes have